x-pack/plugins/security_solution/public/entity_analytics/components/risk_score_enable_section.tsx

```diff
@@ -78,6 +78,27 @@ const RiskScoreErrorPanel = ({ errors }: { errors: string[] }) => (
           ))}
         </>
       </EuiAccordion>
+
+      <EuiAccordion id={'risk-engine-erros'} buttonContent={i18n.CHECK_PRIVILEGES}>
+        <>
+          <p>
+            {i18n.NEED_TO_HAVE}
+            <ul>
+              <li>
+                <b>{'all'}</b> {i18n.PRIVILEGES_FOR} <b>{'risk-score.risk-score-*'}</b> {i18n.INDEX}
```

Comment:
For some languages the word order is different. To give more flexibility to translators we should i18n the entire sentence. 
Ex:
```suggestion
  <FormattedMessage
    id="xpack.securitySolution.myId"
    defaultMessage="{required_privilege} privileges for index"
    values={{    
      // Case we need to translate 'all' we can render <FormattedMessage /> here
      required_privilege: <b>{'all'}</b>, 
      index: <b>{'risk-score.risk-score<'}</b>,
    }}
  />
```
Could you use this pattern for the 3 list items?